src/ui/verificationSymbolStatusView.ts

```diff
@@ -167,33 +167,43 @@ export default class VerificationSymbolStatusView {
     document: TextDocument,
     rootSymbols: DocumentSymbol[] | undefined) {
 
-    this.getVerifiableRangesPromise(params.uri).resolve(params.namedVerifiables.map(v => VerificationSymbolStatusView.convertRange(v.nameRange)));
     this.updateStatusBar(params);
     this.updatesPerFile.set(params.uri, params);
     const controller = this.controller;
-    let items: TestItem[];
+    let leafItems: TestItem[];
     if(rootSymbols !== undefined) {
-      items = this.updateUsingSymbols(params, document, controller, rootSymbols);
+      leafItems = this.updateUsingSymbols(params, document, controller, rootSymbols);
     } else {
-      items = params.namedVerifiables.map(f => VerificationSymbolStatusView.getItem(document,
+      leafItems = params.namedVerifiables.map(f => VerificationSymbolStatusView.getItem(document,
         VerificationSymbolStatusView.convertRange(f.nameRange), controller, document.uri));
-      controller.items.replace(items);
+      controller.items.replace(leafItems);
     }
+    function collectTestItems(collection: TestItemCollection, result: TestItem[]) {
+      collection.forEach(item => {
+        result.push(item);
+        collectTestItems(item.children, result);
+      });
+    }
+    const allTestItems: TestItem[] = [];
+    collectTestItems(controller.items, allTestItems);
```

Comment:
```suggestion
    const allTestItems: TestItem[] = [];
    function collectTestItems(collection: TestItemCollection) {
      collection.forEach(item => {
        allTestItems.push(item);
        collectTestItems(item.children);
      });
    }
    collectTestItems(controller.items);
```
It's more legible with one less argument, unless you plan to lift this function outside of this scope?